### PR TITLE
pipeline_bt_sink: Fixes Static noise for LyraTD-msc Board (AUD-1022)

### PR DESCRIPTION
**Pull request by Tarun-Narain**

### CODE REVIEW

**Comment by Alvin1Zhang** [comment]
@Tarun-Narain Thanks for the contribution.

File: examples/player/pipeline_bt_sink/main/play_bt_music_example.c [hunk 1]
```diff
@@ -118,8 +133,11 @@ void app_main(void)
                      music_info.sample_rates, music_info.bits, music_info.channels);
 
             audio_element_setinfo(i2s_stream_writer, &music_info);
-            i2s_stream_set_clk(i2s_stream_writer, music_info.sample_rates, music_info.bits, music_info.channels);
-            continue;
+#if (CONFIG_ESP_LYRATD_MSC_V2_1_BOARD || CONFIG_ESP_LYRATD_MSC_V2_2_BOARD)
+#else         
+		 i2s_stream_set_clk(i2s_stream_writer, music_info.sample_rates, music_info.bits, music_info.channels);
```
Comment (theronic) [inline examples/player/pipeline_bt_sink/main/play_bt_music_example.c]: i2s_stream_set_clk might be the clincher. Will give it a try.

Comment (Tarun-Narain) [reply examples/player/pipeline_bt_sink/main/play_bt_music_example.c]: Thanks, i'm a big fan of your chess engine!

**Comment by jason-mao** [comment]
@Tarun-Narain Thank you. This MR has been merged on 30403e26485e00c42bb3d8a96064ebcb0090edb8.

**Pull request closed without merging**